packages/slice-machine/lib/models/common/widgets/LinkToMedia/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as yup from 'yup'
+import Form, { FormFields } from './Form'
+import { MdAttachment } from "react-icons/md";
+
+import { handleMockConfig, handleMockContent } from './Mock'
+import { MockConfigForm } from './Mock/Form'
+
+import { Widget } from '../Widget'
+import { FieldType } from '../../CustomType/fields'
+
+import { linkConfigSchema } from "@lib/models/common/widgets/Link";
+import { LinkToMediaField } from "@lib/models/common/widgets/LinkToMedia/type";
+
+const Meta = {
+  icon: MdAttachment,
+  title: 'Link to media',
+  description: 'A link to files, document and media'
+}
+
+const linkToMediaConfigSchema = linkConfigSchema.shape({
+  select: yup.string().required().matches(/^media$/, { excludeEmptyString: true }),
+})
+
+const schema = yup.object().shape({
+  type: yup.string().matches(/^Link$/, { excludeEmptyString: true }).required(),
+  config: linkToMediaConfigSchema.optional()
```

Comment:
there is also a fieldset property described in the custom type API, i don't know if this adds any value here.

Reply:
For now we don't use this data in Link too. Arnaud said that was a old field used as label. I think we can ignore this for now and see if there is some users who are still using this format.